bumped dl version. Add VNA autoscale to data cadence

raphaelcervantes opened the pull request:
Initially, when we take a modemap, s11 wouldn't be visible on the VNA screen. That's because the screen is scaled to fit s21 data, and s11 is larger than s21 by several orders of magnitude. So I can't visually keep track of the reflection measurement while s11 data is being collected.

I made the data cadence autoscale the VNA whenever it switches from s11 and s21. That way, I can always see the data while the measurement is happening. I hope that this helps me understand the experiment better while it's measuring a modemap.

I also upgraded the dripline version.

parasharmohapatra commented:
the_interface.get('na_s21_iq_data')
    #  autoscale the window so that the s21 data fits. This is so I can watch the data while it's being recorded.
    the_interface.set('na_commands', 'autoscale')
do you think you should tell it to sleep for like a second between these 2 commands? This is so that we are sure it has successfully gotten the s21/s11 measurement.

parasharmohapatra commented:
Also we should also add this to the DataLogger class

raphaelcervantes commented:
I have changed the na_s21_iq_data so that it waits for a sweep to finish. So there will be an s21 data before it tries to autoscale
https://github.com/axiondarkmatterexperiment/orpheus_site_atlas/commit/5cdf170b7274ba29c3369f58873612bc32ddad0e

I have also changed the DataLogger class.
